Earthfile

```diff
@@ -83,8 +83,6 @@ docker:
     RUN useradd -m -u 1000 -U -s /bin/sh -d /substrate substrate \
         && mkdir -p /data /substrate/.local/share/partner-chains-node \
         && chown -R substrate:substrate /data /substrate \
-        # remove package managers
-        && rm -rf /usr/bin/apt* /usr/bin/dpkg* \
```

Comment:
Adaptation required for: https://github.com/input-output-hk/partner-chains/pull/402/files#r1925836622